packages/graph/hash_graph/lib/graph/src/store/postgres/query/statement/select.rs

```diff
@@ -213,4 +213,99 @@ mod tests {
             &[],
         );
     }
+
+    #[test]
+    fn property_type_to_data_type() {
+        let mut compiler = SelectCompiler::<PropertyType>::with_asterix();
```

Comment:
This has been fixed in https://github.com/hashintel/hash/pull/1218/commits/31906a75cab97d1ed6335e3211045d100cb87176 but this branch was not rebased yet (will wait for #1230) as this will also affect this PR

Reply:
I rebased on #1230 with this fix applied